generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !python -c "from monai.apps.generation.maisi.utils.morphological_ops import dilate, erode" || pip install git+https://github.com/Project-MONAI/MONAI.git@dev
</pre><p>I don't think we need this one here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Thanks, done